### PR TITLE
feat: automerge lts node versions and keep them in sync

### PR DESCRIPTION
**Pull request by talamcol**
this is how I understand the docs:

`matchManagers`: 
- dockerfile: should look for Dockerfile and there find the docker image
- npm: checks for package.json
- nvm: checks for nvmrc

`versioning: node`
This should only pick LTS versions as far as I understand
image
and also https://docs.renovatebot.com/presets-workarounds/#workaroundstypesnodeversioning in combination with https://github.com/renovatebot/renovate/issues/1463#issuecomment-875562839

Tested it in a test repo and it seems to work fine: 
image

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
       - run: RENOVATE_CONFIG_FILE=web.json npx -p renovate renovate-config-validator
       - run: RENOVATE_CONFIG_FILE=internalSmgPackageUpdate.json npx -p renovate renovate-config-validator
       - run: RENOVATE_CONFIG_FILE=groupDevDependencies.json npx -p renovate renovate-config-validator
-      - run: RENOVATE_CONFIG_FILE=autoMergeNonMajorNode.json npx -p renovate renovate-config-validator
+      - run: RENOVATE_CONFIG_FILE=autoMergeLtsNodeVersions.json npx -p renovate renovate-config-validator
       - run: RENOVATE_CONFIG_FILE=autoMergeNonMajorDocker.json npx -p renovate renovate-config-validator
```
Comment (masone) [inline .circleci/config.yml:32]: seems odd that we need to run the command multiple times with different configs. unrelated to this PR, but maybe we can keep our eyes open to see if there is any way to roll with one execution with multiple configs. or a shared config that references more detailed configs

this kinda is a maintenance issue as we'll have to keep this CI step in sync across multiple projects. simpler is always less prone to errors...

**Review by masone — APPROVED** [review]
thanks for jumping on this. let's see if it works!

**Comment by talamcol** [comment]
seems like it's not working. Going back to the drawing board

**Comment by talamcol** [comment]
@lkappeler as discussed in Slack, I removed the automerge to be on the safe side. We can add it again if we see it works well.

File: .circleci/config.yml (hunk 1 above)
Comment (talamcol) [reply .circleci/config.yml:32]: this step is not in other projects. The config is shared automatically

**Review by lkappeler — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (masone) [reply .circleci/config.yml:32]: mea culpa. I mixed up the context of this change. it's in the renovate config, not a project

**Pull request merged**